lib/libvarnishapi/vut.c

```diff
@@ -336,6 +337,7 @@ VUT_Fini(struct VUT **vutp)
 
 	memset(vut, 0, sizeof *vut);
 	FREE_OBJ(vut);
+	*vutp = NULL;
```

Comment:
NULLing here doesn't close the race with the signal handler. Once it's NULL we may still receive a signal and crash in the `varnishncsa` code (and possibly the other VUTs). We should keep the "take" semantics here and not assert in the signal handler since we have no control over signal delivery in `varnishncsa`'s current state. (We could, but we don't need to.)